plugins/inputs/graylog/graylog.go

```diff
@@ -30,10 +31,11 @@ type Metric struct {
 }
 
 type GrayLog struct {
-	Servers  []string
-	Metrics  []string
-	Username string
-	Password string
+	Servers         []string
+	Metrics         []string
+	Username        string
+	Password        string
+	ResponseTimeout config.Duration
```

Comment:
When you are touching this anyway, can you please add the corresponding `toml` tag annotations?

Reply:
Yes